src/Models/BaseElement.php

```diff
@@ -528,6 +536,85 @@ public function getContentForSearchIndex(): string
         return $content;
     }
 
+    /**
+     * Provides content for CMS search
+     */
+    public function getContentForCmsSearch(): string
+    {
+        $textualDatabaseFields = $this->getTextualDatabaseFields();
+        $excludedFields = $this->getFieldsExcludedFromCmsSearch();
+        $contents = [];
+        foreach ($this->config()->get('db') as $fieldName => $fieldType) {
+            if (!in_array($fieldType, $textualDatabaseFields) || in_array($fieldName, $excludedFields)) {
+                continue;
+            }
+            $contents[] = $this->$fieldName;
```

Comment:
```suggestion
        $content = [];
        foreach ($this->config()->get('db') as $fieldName => $fieldType) {
            if (!in_array($fieldType, $textualDatabaseFields) || in_array($fieldName, $excludedFields)) {
                continue;
            }
            $content[] = $this->$fieldName;
```
This way we reuse the same variable below instead of still holding on to this array of the same content, which we don't need beyond that point.

Reply:
I do not like variable reuse, especially since $contents is an array while $content is a string

Reply:
Not a big deal either way so fine as is